### ISSUE TITLE
CTRL+Right Arrow does not select additional characters past named member attribute syntax 

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#30664]
``` VB
<Assembly: CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.ApplicationDesigner.ApplicationDesignerPanel.CreateDesigner")>
```

1. Highlight 'Justification' and press CTRL+Right Arrow

Expected:
`Justification:` to be selected

Actual:
Nothing, you cannot use CTRL+Right Arrow to select any additional characters

**Comment by sharwell** [comment]
I can't reproduce this in 15.8.8.

* If I press <kbd>Ctrl</kbd>+<kbd>&rarr;<kbd>, the selection goes away and the caret moves one work to the right
* If I press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>&rarr;<kbd>, the caret moves to the right and the selection is extended

**Comment by davkean** [comment]
This _does not_ repro in.
Version 15.8.8

It _does_ repro in:

Version 15.9.0 Preview 4.0 [28215.3051.d15.9stg]
Version 16.0.0 Preview 1.0 [28223.55.d16.0]

**Comment by sharwell** [comment]
@jinujoseph Assigning this to 15.9 milestone to avoid the regression

**Comment by sharwell** [comment]
@jinujoseph This shipped as a regression in 15.9.

**Comment by jinujoseph** [comment]
being investigated by [editor ](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/735705)team
